src/test_workflow/smoke_test/smoke_tests_spec/2.x/sql.yml

```diff
@@ -0,0 +1,159 @@
+---
+info:
+  title: OpenSearch Sql plugin smoke tests
+  version: 2.x
+name: sql
+paths:
+  /_bulk:
+    POST:
+      parameters: [
+        {
+          'index':
+            {
+              '_index':
+                'accounts',
+              '_id':
+                1
+            }
+        },
+        {
+          'account_number':
+            1,
+          'firstname':
+            'Amber',
+          'address':
+            '880 Holmes Lane',
+          'balance':
+            39225,
+          'gender':
+            'M',
+          'city':
+            'Brogan',
+          'employer':
+            'Pyrami',
+          'state':
+            'IL',
+          'age':
+            32,
+          'email':
+            'amberduke@pyrami.com',
+          'lastname':
+            'Duke'
+        },
+        {
+          'index':
+            {
+              '_index':
+                'accounts',
+              '_id':
+                6
+            }
+        },
+        {
+          'account_number':
+            6,
+          'firstname':
+            'Hattie',
+          'address':
+            '671 Bristol Street',
+          'balance':
+            5686,
+          'gender':
+            'M',
+          'city':
+            'Dante',
+          'employer':
+            'Netagy',
+          'state':
+            'TN',
+          'age':
+            36,
+          'email':
+            'hattiebond@netagy.com',
+          'lastname':
+            'Bond'
+        },
+        {
+          'index':
+            {
+              '_index':
+                'accounts',
+              '_id':
+                13
+            }
+        },
+        {
+          'account_number':
+            13,
+          'firstname':
+            'Nanette',
+          'address':
+            '789 Madison Street',
+          'balance':
+            32838,
+          'gender':
+            'F',
+          'city':
+            'Nogal',
+          'employer':
+            'Quility',
+          'state':
+            'VA',
+          'age':
+            28,
+          'email':
+            'nanettebates@quility.com',
+          'lastname':
+            'Bates'
+        },
+        {
+          'index':
+            {
+              '_index':
+                'accounts',
+              '_id':
+                18
+            }
+        },
+        {
+          'account_number':
+            18,
+          'firstname':
+            'Dale',
+          'address':
+            '467 Hutchinson Court',
+          'balance':
+            4180,
+          'gender':
+            'M',
+          'city':
+            'Orick',
+          'employer':
+            null,
+          'state':
+            'MD',
+          'age':
+            33,
+          'email':
+            'daleadams@boink.com',
+          'lastname':
+            'Adams'
+        }
+      ]
+  /_plugins/_sql:
```

Comment:
FYI, here is the documentation for `ppl` endpoints which are also essential for sql plugin. https://opensearch.org/docs/latest/search-plugins/sql/sql-ppl-api/#sample-explain-request-for-a-ppl-query

Reply:
Hey @RyanL1997 thanks for suggestions. I have added corresponding `ppl` tests. Please help take another look. Thanks!

Reply:
LGTM. Thanks